rust/crd/src/catalog/delta_lake.rs

```diff
@@ -1,7 +1,7 @@
 use super::commons::{HdfsConnection, MetastoreConnection};
 use serde::{Deserialize, Serialize};
 use stackable_operator::{
-    commons::s3::S3ConnectionDef,
+    commons::s3::S3ConnectionInlineOrReference,
```

Comment:
thought: The name of the struct is tied to the variants inside which are not guaranteed to change.

_I know this should have been brought up earlier. Nothing to change, but just communicating thought_.

Reply:
Makes sense, thanks!